spec/controllers/rentals_controller_spec.rb

```diff
@@ -247,19 +247,31 @@
 
   describe 'GET #search_users' do
     before(:each) do
-      User.destroy_all
+      create_list :user, 20
     end
 
-    it 'returns everything if cant find a user' do # dynamically searches by email, spire, fullname and department (in that order)
+    it 'returns nothing if cant find a user' do # dynamically searches by email, spire, fullname and department (in that order)
       create_list :user, 8 # can only do 8 because of pagination
       get :search_users, params: { user_search_query: '$%%!$#' } # this query should return no users
       expect(assigns[:users]).to be_empty
     end
 
-    it 'finds by spire' do
-      this_one = create :user, spire_id: 86_753_091 # rubocop demanded i separate this huggeeee number with underscores (which are ignored)
-      get :search_users, params: { user_search_query: '86753091' } # should be unique enough
-      expect(assigns[:users]).to contain_exactly(this_one)
+    context 'finds by spire' do
+      before(:each) do
+        # rubocop demanded i separate this huggeeee number with underscores (which are ignored)
+        @this_one = create :user, spire_id: 86_753_091
```

Comment:
I think it'd be a little simpler to just make the spire IDs strings to begin with, then we can remove these funky underscores and the rubocop comment.